src/components/fpl/MatchResults.tsx

```
@@ -320,611 +371,534 @@ export default function MatchResults({
+                                  className="flex items-center justify-between p-3 bg-theme-card rounded-md border-theme-border hover:shadow-md transition-all cursor-pointer theme-transition"
+                                  onClick={() => {
+                                    if (
+                                      player.ownership_top10k &&
+                                      player.ownership_top10k > 5
+                                    ) {
+                                      setSelectedManagerId(player.id);
+                                      onManagerSelect?.(player.id);
+                                    }
+                                  }}
+                                >
+                                  <div className="flex items-center gap-3">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)